### ISSUE TITLE
kci_build: Support defconfigs from Chromeos

### ISSUE THREAD

**Issue by ribalda** [kernelci/kernelci-core#712]
ChromeOS defines its defconfigs as "flavours" in the kernel repository. (Eg: https://chromium.googlesource.com/chromiumos/third_party/kernel/+/refs/heads/chromeos-5.10/chromeos/config/ )

Allow kci_build to create defconfigs base on ChromeOS flavours to have configs like:

```yaml
 next:
    tree: next
    branch: 'master'
    variants:
      chromiumos-qualcomm:
        build_environment: clang-12
        arm64: *arm64_arch
        base_defconfig: 'cros://chromeos-5.10/config/arm64/chromiumos-qualcomm.flavour.config' 
```

**Comment by gctucker** [comment]
> ```yaml
>         base_defconfig: 'cros://chromeos-5.10/config/arm64/chromiumos-qualcomm.flavour.config' 
> ```

Is the `cros://` protocol documented anywhere?

**Comment by ribalda** [comment]
It is not a real network protocol, just a naming convention.

**Comment by gctucker** [comment]
@ribalda We should be able to close this now that #713 has been merged, unless you want to wait for #729 to be merged too?

**Comment by gctucker** [comment]
All merged now, closing this issue.
